src/transforms/group.js

```diff
@@ -266,6 +270,16 @@ const reduceFirst = {
   }
 };
 
+const reduceTitle = {
+  reduce(I, X) {
+    return sort(rollup(I, V => V.length, i => X[i]), second)
+      .reverse()
+      .slice(0, 5)
+      .map(([key, value]) => `${key} (${value.toLocaleString("en-US")})`)
+      .join("\n");
+  }
```

Comment:
```suggestion
    const labels = sort(rollup(I, V => V.length, i => X[i]), second)
      .reverse();
    return labels
      .slice(0, 5)
      .map(([key, value], i) => i === 4 && labels.length > 5 ? "…" : `${key} (${value.toLocaleString("en-US")})`)
      .join("\n");
  }
```

to replace the 5th element by continuation dots if there are more?

Reply:
Probably worth counting the number of “other” then.

Reply:
If there are more than five distinct values, these are now combined into an “Other” category.